content/applications/marketing/events/registration_desk.rst

```diff
@@ -0,0 +1,106 @@
+used. Odoo **must** be granted access to the camera, in order for the :guilabel:`Scan a badge`
+option to work.
+
+Once Odoo has access to the camera, and :guilabel:`Scan a badge` is clicked, a :guilabel:`Barcode
+Scanner` pop-up window, showing the camera's point-of-view. There is also a specified view finder
+box present, which can be enlarged or shrunk, accordingly, using the :icon:`fa-crop`
+:guilabel:`(crop)` icon.
+
+.. image:: registration_desk/barcode-scanner-window.png
+   :align: center
+   :alt: The Barcode Scanner window of the Registration Desk in the Odoo Events application.
+
+Once the badge code is in the middle of the view finder window, the code is scanned, the
+:guilabel:`Barcode Scanner` pop-up window disappears, and the attendee is granted access to the
+event. Once the code is scanned, their attendance is logged in the Odoo *Events* app.
+
+If the barcode being scanned is invalid, an error pop-up message appears in the upper-right corner.
+
+Select attendee
+===============
+
+If users wish to manually grant access to event attendees as they show up to the event, they would
+navigate to the :guilabel:`Registration Desk` page (:menuselection:`Events app --> Registration
+Desk`), and click the :guilabel:`Select Attendee` option.
+
+Upon doing so, Odoo reveals a separate :guilabel:`Attendees` page, showcasing all the attendees for
+every event in the database, in a default :icon:`oi-view-kanban` :guilabel:`Kanban` view.
+
+.. image:: registration_desk/attendees-page.png
+   :align: center
+   :alt: The Attendees page, via the Registration Desk, located in the Odoo Events application.
+
+.. note::
+   The :guilabel:`Attendees` page can also be viewed as a :icon:`oi-view-list` :guilabel:`List`,
+   :icon:`fa-calendar` :guilabel:`Calendar`, or :icon:`fa-area-chart` :guilabel:`Graph`. Each view
+   option is accessible, via their respective icons, located in the upper-right corner of the
+   :guilabel:`Attendees` page.
+
+On the :guilabel:`Attendees` page, in the default :icon:`oi-view-kanban` :guilabel:`Kanban` view,
+each attendee card displays that person's name, which event they are registered to attend, their
+associated company (if applicable), what ticket tier they purchased (if applicable), along with two
+buttons: a :icon:`fa-check` :guilabel:`(checkmark)` and :icon:`fa-undo`
+:guilabel:`(counter-clockwise arrow)`.
+
+To grant access to a person, and officially mark them as attended, click the :icon:`fa-check`
+:guilabel:`(checkmark)` button on that specific attendee's card.
+
+If a mistake was made, or if the incorrect attendee was marked as attended, click the
+:icon:`fa-undo` :guilabel:`(counter-clockwise arrow)` button on that specific attendee's card.
+
+It is recommended to use an event-specific filter on the :guilabel:`Attendees` page, via the
+:guilabel:`Search...` bar, to avoid having to search through a master list of attendees related to
+every event in the database.
```

Comment:
```suggestion
Use an event-specific filter on the :guilabel:`Attendees` page, via the
:guilabel:`Search...` bar, to avoid searching through a master list of all event attendees.
```

Reply:
In my opinion, posing this as a recommendation seems more appropriate, as it is *technically* not necessary for them to complete the task. But I fully understand (and appreciate) the suggestion to make things a bit more concise.